src/browser/modules/Sidebar/Documents.tsx

```diff
@@ -142,12 +152,17 @@ const Documents = ({ version, urlVersion }: DocumentsProps) => {
     }
   }, [])
 
-  const { docs, other } = getReferences(version, urlVersion)
+  const { docs, other } = getReferences(props.version, props.urlVersion)
   return (
     <Drawer id="db-documents">
       <StyledHeaderContainer>
         <DrawerHeader>Help &amp; Learn</DrawerHeader>
-        <a data-canny-changelog>
+        <a
+          data-canny-changelog
+          onClick={() => {
+            props.trackCannyChangelog()
```

Comment:
this could be written: onClick={props.trackCannyChangelog}, just a personal preference